### DISCUSSION TITLE
Windows Pre-Built Binaries

### DISCUSSION THREAD

**Discussion by BrianBMN** [coinfoundry/miningcore#1030] [Q&A] [answered]
I couldn't seem to find them, but I might blind. Are they available or is building the only option now?

I installed the SDK as outlined in the readme as well as git but I get the following when trying to build. I am not a dev, rather a systems engineer, so this is somewhat greek to me :D 

![image](https://user-images.githubusercontent.com/3384812/145183984-57e88cae-5f89-4bc7-9cc4-2fff80363682.png)

**Comment by oliverw** [comment, marked as answer]
The introduction of Gitversion into the project made it impossible to build without having the .git history present in your build folder. In short you have to clone the repo instead downloading a .zip. I need to implement some workaround for this.

**Reply by BrianBMN** [reply]
Build successful.
